safe_transaction_service/history/tasks.py

```diff
@@ -57,20 +57,18 @@
 @app.shared_task(bind=True, soft_time_limit=SOFT_TIMEOUT, time_limit=LOCK_TIMEOUT)
 def check_reorgs_task(self) -> Optional[int]:
     """
-    :return: Number of oldest block with reorg detected. `None` if not reorg found
+    :return: Number of the oldest block with reorg detected. `None` if not reorg found
     """
     with contextlib.suppress(LockError):
         with only_one_running_task(self):
             logger.info("Start checking of reorgs")
             reorg_service: ReorgService = ReorgServiceProvider()
-            first_reorg_block_number = reorg_service.check_reorgs()
-            if first_reorg_block_number:
-                logger.warning(
-                    "Reorg found for block-number=%d", first_reorg_block_number
-                )
+            reorg_block_number = reorg_service.check_reorgs()
+            if reorg_block_number:
+                logger.warning("Reorg found for block-number=%d", reorg_block_number)
                 # Stopping running tasks is not possible with gevent
-                reorg_service.recover_from_reorg(first_reorg_block_number)
-                return first_reorg_block_number
+                reorg_service.recover_from_reorg(reorg_block_number)
```

Comment:
`recover_from_reorg` receive a parameter called `first_reorg_block_number`. Maybe we could keep `first_reorg_block_number` or `reorg_block_number` in both places.

Reply:
You are right. Done, can you recheck?